src/containers/applications/tabs2.scss

```diff
@@ -93,16 +92,16 @@ rediv {
 
   &[data-err="true"] {
     .oper {
-      color: #aaa;
+      color: var(--dark-txt);
```

Comment:
make a global --comp-gray-txt with value ```aaa``` in light mode and ```555``` in dark mode

Reply:
this is because the button get disabled when an invalid operation is operated (like dividing by zero)

Reply:
in win11 it is #ffffff only (i checked it)

![image](https://user-images.githubusercontent.com/89068816/140053016-8b7fe505-a656-401b-b5ab-f84edb12a6bc.png)

Reply:
I know but I programmed it with keeping it simple and i.e after invalid operation you can only press C button and restart equation

Reply:
maybe you can add that clicking anything 0-9 , CE , Backspace & = also restart the equation